File: someerre/someerre/ViewController.m
//
//  ViewController.m
//  someerre
//
//  Created by kuutsav on 6/9/17.
//  Copyright © 2017 kuutsav. All rights reserved.
//

#import "ViewController.h"
#import <CPEModels/CPEModels.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    BaseClass *bclass = [[BaseClass alloc]init];
    PxStageHistory *pStageHistory  = [[PxStageHistory alloc]init];
    PxFlow *pflow = [[PxFlow alloc]init];
    
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: se2/se2/ViewController.swift
//
//  ViewController.swift
//  se2
//
//  Created by kuutsav on 5/31/17.
//  Copyright © 2017 kuutsav. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var someview:SomeView! = SomeView()
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Do any additional setup after loading the view, typically from a nib.
        
      self.view.addSubview(someview)
        
        
        
    }

    

}

